app/models/user.rb

```diff
@@ -156,6 +156,31 @@ def mark_in_person_completion_survey_sent(issuer)
     nil
   end
 
+  def increment_second_factor_attempts_count!
+    User.transaction do
+      sql = <<-SQL
+      UPDATE users
+      SET second_factor_attempts_count = COALESCE(second_factor_attempts_count, 0) + 1,
+      updated_at = NOW(),
+      second_factor_locked_at = CASE
+        WHEN COALESCE(second_factor_attempts_count, 0) + 1 >= ?
+        THEN NOW()
+        ELSE NULL
+        END WHERE id = ? RETURNING second_factor_attempts_count, second_factor_locked_at;
+      SQL
```

Comment:
small requests to
1. indent the query to make vertical skimming easier
2. put `RETURNING ` on its own line
3. 
```suggestion
      sql = <<~SQL
        UPDATE users
        SET
          second_factor_attempts_count = COALESCE(second_factor_attempts_count, 0) + 1,
          updated_at = NOW(),
          second_factor_locked_at = CASE
            WHEN COALESCE(second_factor_attempts_count, 0) + 1 >= ?
            THEN NOW()
            ELSE NULL
            END
        WHERE id = ?
        RETURNING second_factor_attempts_count, second_factor_locked_at;
      SQL
```